$OpenBSD: patch-widgets_hashBox_cpp,v 1.3 2015/09/02 15:19:28 sthen Exp $

if updating, check they haven't added any more entries to hashalgos[]

--- widgets/hashBox.cpp.orig	Sat Mar 21 08:08:29 2015
+++ widgets/hashBox.cpp	Wed Sep  2 16:13:54 2015
@@ -26,8 +26,8 @@ QString hashBox::default_md = QString();
 
 void hashBox::resetDefault()
 {
-	/* SHA1 */
-	default_md = QString(hashalgos[1].name);
+	/* SHA256 */
+	default_md = QString(hashalgos[3].name);
 }
 
 hashBox::hashBox(QWidget *parent)
